samples/client/petstore/erlang-client/.openapi-generator/VERSION

```diff
@@ -1 +1 @@
-6.5.0-SNAPSHOT
\ No newline at end of file
+5.3.0-SNAPSHOT
```

Comment:
looks like your fix is not based on the latest master. i'll update the samples after merging this PR.

Reply:
🤔 hrm I thought I had branched off of the latest master.i is there something I should normally do when switching to the master branch to make sure the correct version is used normally?